operator/internal/manifests/storage/configure.go

```diff
@@ -116,6 +126,22 @@ func configureStatefulSetCA(s *appsv1.StatefulSet, tls *TLSConfig) error {
 	return nil
 }
 
+func ensureObjectStoreStructuredMetadata(p *corev1.PodSpec, opts Options) corev1.PodSpec {
+	if !opts.AllowStructuredMetadata() {
+		return *p
+	}
+
+	container := p.Containers[0].DeepCopy()
+
+	container.Args = append(container.Args, "-validation.allow-structured-metadata=false")
```

Comment:
Shouldn't it be `true`  when we want to allow structured metadata? 
```suggestion
	container.Args = append(container.Args, "-validation.allow-structured-metadata=true")
```

Reply:
This does not enable or disable the structured metadata itself, but only disables the _validation_.

Reply:
True don't know why I didn't open the link you provided in the other comment... 🤦 

Plus this documentation is misleading https://grafana.com/docs/loki/latest/send-data/otel/#loki-configuration